src/test/java/com/thealgorithms/datastructures/lists/ReverseKGroupTest.java

```diff
@@ -4,10 +4,7 @@
 import static org.junit.jupiter.api.Assertions.assertNull;
 
 import org.junit.jupiter.api.Test;
-/**
- * Test cases for Reverse K Group LinkedList
- * Author: Bama Charan Chhandogi (https://github.com/BamaCharanChhandogi)
- */
```

Comment:
Why did you remove this?

Reply:
I was removing excessive comments from the test code, the above line was deleted by mistake. I added it back